### ISSUE TITLE
max num of insert 

### ISSUE THREAD

**Issue by yanyumissing** [Impetus/kundera#7]
I use Kundera 2.0.1 to build a simple project with only one CF.
For the insert method, it passed the Junit test.
But when I try to insert n rows, it seems to be a trouble

the test:
protected void executeTestSuite()
            {
                Integer i;
                for (i=0; i<N; i++){
                    addUsers();

```
            }

        }
```

the implementation:
public void create(//ommitted) {

```
            SnsUser snsuser = new SnsUser(snsuid, blessuid, snstype, localId, accountName);
        em.persist(snsuser);

    }
```

N=1  pass
N=100 pass
N=200 pass
N=257 pass but lost some rows, only get 27rows insert
N=1000 Server crushed

Have you ever done such kind of test on Kundera?

best regards!

**Comment by mevivs** [comment]
Not yet tried. Can you please share errors which you have got on server cash.?

**Comment by mevivs** [comment]
Hi Yanu,
In case if you could share your logs can help us to resolve this. We are currently analyzing performance of Kundera.

I guess, you would be testing it with 1 node.

**Comment by yanyumissing** [comment]
Yes I tested it with one node.
In my test case, when the N>200 (insert rows in one testcase), the server
blocked without any error log. I forced stop it and restart the server, some
of the rows are inserted in the cassandra database (not stable number each
time).
Further more, before stop the server, I try to restart the client side, with
command connect to localhost:9165, the response is: connect refused, no
cache memory on the port.
BTW, N=100 is ok for every time.
I haven't change any configuration on both cassandra and solandra (according
to the kundera example)

2011/8/4 mevivs <
reply@reply.github.com>

**Comment by mevivs** [comment]
Will have a look on it. I need to explore and build in support for batch update with Cassandra. Taking this on prioirity.

**Comment by mevivs** [comment]
Did you try changing settings like

rpc_min_threads: 16
rpc_max_threads: 2048

in cassandra.yaml?

I am still exploring to resolve this. But outofMemory error i am getting is because of cassandra server itself.

**Comment by mevivs** [comment]
Hi,
Just to update u guys that a new section is created for Kundera cassandra integration trouble shooting at

Cassandra specific configuration

https://github.com/impetus-opensource/Kundera/wiki/Download-Kundera

**Comment by mevivs** [comment]
This is fixed in 2.0.4 release and testing with 1 million insert on single as well concurrent threads.
